raiden-ts/tests/e2e/e2e.spec.ts

```diff
@@ -60,6 +60,9 @@ async function createRaiden(account: number | string | Signer): Promise<Raiden>
       confirmationBlocks: 5,
       autoSettle: false, // required to use `settleChannel` later
       autoUDCWithdraw: false, // required to use `withdrawFromUDC` later
+      caps: {
+        [Capabilities.RECEIVE]: 1,
+      }
```

Comment:
Linter is complaining
```suggestion
      },
```